client/app/components/Modal/__tests__/Modal.spec.jsx

```diff
@@ -43,15 +43,23 @@ describe('Modal', () => {
     describe('has text values for element and body', () => {
       const component = <Modal element="Hello" body={bodyText} title={title} />;
       it('toggles correctly', () => {
-        const wrapper = mount(component);
-        expect(wrapper.find('.modalBackdrop').exists()).toEqual(false);
-        expect(wrapper.find('.modal').exists()).toEqual(false);
-        wrapper.find('.modalElement').simulate('click');
-        expect(wrapper.find('.modalBackdrop').exists()).toEqual(true);
-        expect(wrapper.find('.modal').exists()).toEqual(true);
-        wrapper.find('.modalClose').simulate('click');
-        expect(wrapper.find('.modalBackdrop').exists()).toEqual(false);
-        expect(wrapper.find('.modal').exists()).toEqual(false);
+        const { container, queryByLabelText, queryByRole } = render(component);
```

Comment:
Let's use `screen.queryByLabelText` and `screen.queryByRole` by importing `screen` from `@testing-library/react` instead.